bin/rspecq

```diff
@@ -83,6 +89,7 @@ opts[:timings] ||= env_set?("RSPECQ_UPDATE_TIMINGS")
 opts[:file_split_threshold] ||= Integer(ENV["RSPECQ_FILE_SPLIT_THRESHOLD"] || 9999999)
 opts[:report] ||= env_set?("RSPECQ_REPORT")
 opts[:report_timeout] ||= Integer(ENV["RSPECQ_REPORT_TIMEOUT"] || DEFAULT_REPORT_TIMEOUT)
+opts[:max_requeues] ||= Integer(ENV["MAX_REQUEUES"] || DEFAULT_MAX_REQUEUES)
```

Comment:
We'd better follow the convention here too and prepend the environment variable with "RSPECQ_", to avoid conflict with other, non-RSpecQ variables.